contents/javascript/VueJs/Intro.md

```diff
@@ -0,0 +1,85 @@
+# VueJs
+
+Author: Lu Lechuan
+
+A JavaScript framework is an application framework written in JavaScript. It differs from a JavaScript library in its control flow: a library offers functions to be called by its parent code, whereas a framework defines the entire application design.
```

Comment:
I think first statement should be about `What is VueJS` rather than explaining what is difference between library and framework.

Reply:
Previously I think if I starts with what is VueJs - "a (some adjectives) JavaScript framework", I leave JavaScript framework as an undefined term. Do you think it is fine?

Reply:
I think that's good, maybe add link to JavaScript framework referring to external resources so that they can know what a Javascript framework is from there

Reply:
Ok sure!